From 49ca5b62dd6d95b78c1086e400ae42f410bde30b Mon Sep 17 00:00:00 2001
From: Andy Yan <andy.yan@rock-chips.com>
Date: Fri, 11 Dec 2020 19:20:11 +0800
Subject: [PATCH] arm64: dts: rockchip: assign DCLK_VOP1 to VPLL for rk3568 nvr

NVR demo board use DCLK_VOP1 drive VGA monitor, which should
as accuracy as possible.

Change-Id: Ie19174cf2b0c951216c8f5e1850758a9df66be0b
Signed-off-by: Andy Yan <andy.yan@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3568-nvr.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3568-nvr.dtsi b/arch/arm64/boot/dts/rockchip/rk3568-nvr.dtsi
index 32413bd7b71a..c9f8d59005ac 100644
--- a/arch/arm64/boot/dts/rockchip/rk3568-nvr.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3568-nvr.dtsi
@@ -449,6 +449,8 @@
 
 &vop {
 	status = "okay";
+	assigned-clocks = <&cru DCLK_VOP1>;
+	assigned-clock-parents = <&cru PLL_VPLL>;
 };
 
 &vop_mmu {
-- 
2.35.3

